service/matching/service.go

```diff
@@ -22,76 +22,71 @@ package matching
 
 import (
 	"github.com/uber/cadence/common"
-	"github.com/uber/cadence/common/log/loggerimpl"
+	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/log/tag"
-	"github.com/uber/cadence/common/persistence/client"
+	"github.com/uber/cadence/common/persistence"
+	persistenceClient "github.com/uber/cadence/common/persistence/client"
+	"github.com/uber/cadence/common/resource"
 	"github.com/uber/cadence/common/service"
 	"github.com/uber/cadence/common/service/dynamicconfig"
 )
 
 // Service represents the cadence-matching service
 type Service struct {
-	stopC  chan struct{}
-	params *service.BootstrapParams
+	resource.Resource
```

Comment:
is ServiceCommon/ServiceBase a better name?

Reply:
i will change this resource thing to service after the moving everything, then delete service, then renaming it to service